package.json

```diff
@@ -16,7 +16,7 @@
   },
   "categories": ["Other"],
   "activationEvents": ["onCommand:center-editor-window.center"],
-  "main": "./out/src/extension",
+  "main": "./out/extension",
```

Comment:
is this line intended?

Reply:
The extension would not run on my computer without it. I'm not sure about the author.

Reply:
It looks like there was something changed in the VScode output path. When I originally generated the extension template, the "src" path was standard. My first transpiled ext was still in that folder - and everything works.

From what I learned, the default nowadays is not having "src" in the path. So thanks, I'll take it, even if its not related to the PR ;)